app/views/contexts/thank_you_context.py

```diff
@@ -19,5 +26,14 @@ def build_default_thank_you_context(session_data):
     return context
 
 
-def build_census_thank_you_context(display_address, census_type):
-    return {"display_address": display_address, "census_type": census_type}
+def build_census_thank_you_context(
+    session_data: SessionData, census_type_code: str
+) -> Mapping:
+    return {
+        "display_address": session_data.display_address,
+        "census_type": census_type_code,
+        "hide_signout_button": False,
+        "log_out_url": url_for(
+            "session.get_sign_out", log_out_url=get_census_base_url()
```

Comment:
Would this `log_out_url` be better named as something else since it is just redirecting?. `redirect_url` etc?

Also, should we pass the redirect url to the templates, or template just passes and arg which determines in code where to go?.

Reply:
`redirect_url` is more apt.

As for passing to templates, or doing it in code, I think we keep it as is, otherwise we begin to pollute the session module, which I was keen to avoid